.github/workflows/ci.yml

```diff
@@ -46,12 +47,10 @@ jobs:
 
       - name: Prepare
         run: |
-          sudo apt install fd-find
+          sudo apt update
+          sudo apt install fd-find --yes
           test -d build || {
-            mkdir -p build
-            wget https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage
-            chmod +x nvim.appimage
-            mv nvim.appimage ./build/nvim
+            bash .github/workflows/scripts/setup-appimage.bash
```

Comment:
you already have `bash` in the shebang? why not make the script executable and do `./.github/workflows/scripts/setup-appimage.bash`

i also prefer scripts to end on `.sh`

Reply:
Sure! I'll do this when I get home